projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DataJobMetrics.java

```diff
@@ -173,6 +172,33 @@ public void updateTerminationStatusGauge(final DataJob dataJob) {
         }
     }
 
+    /**
+     * Creates a "taurus.datajob.termination.status.counter" counter for the specified
+     * data job and execution status, if one does not exist. If a counter already
+     * exists, its value is incremented.
+     *
+     * @param dataJob The data job for which to increment the counter.
+     * */
+    public void incrementTerminationStatusCounter(final DataJob dataJob) {
```

Comment:
Looking at the PR description

> This change introduces "counter" metrics that measure the number of specific statuses
> observed for each data job to help with situations when high-level picture of data job
> executions is needed.

How practical would that change be? If we monitor the count only, then how do we establish what data jobs do indicate those specific statuses?

Reply:
The change is more for statistic purposes. It would allow the user to get a high-level picture of the overall job executions. Also, because the counter has label for job_name and status, one could possibly have a query to filter by job_name.